src/common/exif.cc

```diff
@@ -1196,8 +1193,34 @@ int dt_exif_read_blob(uint8_t **buf, const char *path, const int imgid, const in
 
       {
         static const char *keys[] = {
+          // Color profile info
+          "Exif.Image.CalibrationIlluminant1",
+          "Exif.Image.CalibrationIlluminant2",
+          "Exif.Image.ColorMatrix1",
+          "Exif.Image.ColorMatrix2",
+          "Exif.Image.ForwardMatrix1",
+          "Exif.Image.ForwardMatrix2",
+          "Exif.Image.ProfileCalibrationSignature",
+          "Exif.Image.ProfileCopyright",
+          "Exif.Image.ProfileEmbedPolicy",
+          "Exif.Image.ProfileHueSatMapData1",
+          "Exif.Image.ProfileHueSatMapData2",
+          "Exif.Image.ProfileHueSatMapDims",
+          "Exif.Image.ProfileLookTableData",
+          "Exif.Image.ProfileLookTableDims",
+          "Exif.Image.ProfileName",
+          "Exif.Image.ProfileToneCurve",
+          "Exif.Image.ReductionMatrix1",
+          "Exif.Image.ReductionMatrix2",
+          // Some DNG tags not yet registered in exiv2
+          //"Exif.Image.BaselineExposureOffset", // 0xc7a5
```

Comment:
Actually, we do not know which exiv2 version the darktable will be linked against.
It might now know about some of the tags listed here.
So it is safe to list any string here, but the code that deletes the tags should be safeguarded to not fail on unknown tags.